templates/blog/post_list.html

```diff
@@ -0,0 +1,21 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Posts</h1>
+    {% for post in post_list%}
+      <div class="post">
+        <h4 class="title">
+          <a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a>
+        </h4>
+        <p>{{post.content}}</p>
```

Comment:
Prefer adding blank spaces at the beginning and at the end of the Jinja variable.